test/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -31,6 +31,8 @@ internal abstract class MaestroScenarioTestBase
 {
     private TestParameters _parameters = null!;
     private List<string> _baseDarcRunArgs = [];
+    // We need this for tests where we have multiple updates
+    private Dictionary<long, DateTimeOffset> _lastUpdatedPrTimes = new();
```

Comment:
I went with a dictionary here because I'm not sure if we sometimes have multiple tests at the same time

Reply:
So this was the safest option